hack/lib/catalogsource.bash

```diff
@@ -73,8 +73,9 @@ apiVersion: operators.coreos.com/v1alpha1
 kind: CatalogSource
 metadata:
   name: ${OPERATOR}
+  namespace: openshift-marketplace
```

Comment:
We modified some code here, that by default it will take empty and used the existing build methods, if any parameter passed it will try to update the images
  if [ ! -n "${IMAGE_INDEX_BUNDLE:-}" ]; then
      index_image="$IMAGE_INDEX_BUNDLE"
  fi